hovercraft/local_directives/__init__.py

```diff
@@ -0,0 +1,40 @@
+import os
```

Comment:
It's better to not use __init__ files for a lot of code. I'm transitioning away from that,